client/dashboard/sites/overview/site-card.tsx

```diff
@@ -8,23 +9,24 @@ import {
 } from '@wordpress/components';
 import { dateI18n } from '@wordpress/date';
 import { __, sprintf } from '@wordpress/i18n';
+import { sitePHPVersionQuery } from '../../app/queries';
 import { getSiteStatusLabel } from '../../utils/site-status';
 import SitePreview from '../site-preview';
 import type { Site, Plan } from '../../data/types';
 
+function PHPVersion( { siteSlug }: { siteSlug: string } ) {
+	return (
+		( useQuery( sitePHPVersionQuery( siteSlug ) ).data && null ) ?? (
+			<span className="text-blur" data-text="X.Y" />
```

Comment:
This is interesting approach... might be beneficial to extract this into a reusable placeholder loading component (or we might already have one?)

Reply:
If we plan to reuse, yes! :)

Reply:
The classname is not correct here, it doesn't follow our guidelines.

Reply:
What are the guidelines? And why are there no tests?

Reply:
https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md?term=css%20guidelines

It's basically the same guidelines as Gutenberg.

Reply:
You can tell how much I touch CSS. It would be good to have some linter for this.